seatunnel-server/pom.xml

```diff
@@ -27,6 +27,17 @@
     <packaging>pom</packaging>
     <modules>
         <module>seatunnel-app</module>
+        <module>seatunnel-spi</module>
```

Comment:
Can we rename `seatunnel-spi` to `seatunnal-scheduler-spi`? It seems this module only defines some interfaces for the scheduler.

Reply:
It is necessary not to change to 'scheduler-spi', and our script may have multiple implementations in the future